src/test/java/Page_Objects/SongListPage.java

```diff
@@ -124,4 +126,30 @@ public void approveSongByKeyboard() {
     public boolean isSongMarkedWithDuetIcon(String songID) {
         return getSongElement(songID).findElement(By.cssSelector("[data-test='multitrack-indicator']")).isDisplayed();
     }
+
+    public void expectSongToBeMarkedAsPlayedToday(String songID) {
+        String playedStatusText = getSongElement(songID).findElement(By.cssSelector("[data-test='song-stat-indicator']")).getText();
+        String playedStatusSubs = playedStatusText.substring(0, 1).toUpperCase() + playedStatusText.substring(1).toLowerCase();
+
+        Assertions.assertEquals("Played today", playedStatusSubs, "Played today status should be visible for the song");

```

Comment:
similar to the above comment about the error message, you could include the song id and what is the actual text here